pkg/scanner/local/scan.go

```diff
@@ -261,6 +261,18 @@ func (s Scanner) scanLicenses(target types.ScanTarget, options types.ScanOptions
 	var osPkgLicenses []types.DetectedLicense
 	for _, pkg := range target.Packages {
 		for _, license := range pkg.Licenses {
+			if strings.HasPrefix(license, "text://") {
+				osPkgLicenses = append(osPkgLicenses, types.DetectedLicense{
+					Severity:    dbTypes.SeverityUnknown.String(),
+					Category:    ftypes.CategoryUnknown,
+					PkgName:     pkg.Name,
+					Name:        "CUSTOM License",
```

Comment:
We need to add info when we use `CUSTOM License` in docs and `license Name`/`log message`

Reply:
We might want to add the first few words of the license text to the `name`.

Reply:
> We might want to add the first few words of the license text to the `name`.

if @knqyf263 agrees too, I'll add it.
I thought about filtering by `CUSTOM License`, but maybe it doesn't matter

Reply:
@DmitriyLewen  ))
![изображение](https://github.com/user-attachments/assets/a2a75022-4b2c-4024-b145-7a213693a8e4)

Reply:
I thought about `CUSTOM License` again. This may not be clear to users.
What if we use the name `Incomparable License`/`Unmatched License`
This means that we can't compare license text with known Trivy licenses and therefore store license as text.
cc. @knqyf263

Reply:
I saw some places calling them "custom licenses", so I don't think it's that bad.

>Custom licenses are strongly disfavored and should be used only in extraordinary circumstances. An Agency ought to adopt a license approved by the Open Source Initiative (OSI) at opensource.org. 

https://opensource.org/authority

>If you are using a license that hasn't been assigned an SPDX identifier, or if you are using a custom license, use a string value like this one:

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#license

>Alternatively, you can use a LicenseRef- custom license identifier to refer to a license that is not on the SPDX License List, such as the following:

https://spdx.github.io/spdx-spec/v2.3/using-SPDX-short-identifiers-in-source-files/

Or "non-standard" or something like that.

`Incomparable License`/`Unmatched License` are also unclear what these did not match.